InternTrack.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.Plans.cs

```diff
@@ -3,26 +3,45 @@
 // FREE TO USE FOR THE WORLD
 // -------------------------------------------------------
 
-using Microsoft.Azure.Management.AppService.Fluent;
-using Microsoft.Azure.Management.ResourceManager.Fluent;
-using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
-using  OperatingSystem = Microsoft.Azure.Management.AppService.Fluent.OperatingSystem;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.AppService;
+using Azure.ResourceManager.AppService.Models;
+using Azure.ResourceManager.Resources;
 
 namespace InternTrack.Core.Api.Infrastructure.Provision.Brokers.Clouds
 {
     public partial class CloudBroker
     {
-        public async ValueTask<IAppServicePlan> CreatePlanAsync(
+        public async ValueTask<AppServicePlanResource> CreatePlanAsync(
             string planName,
-            IResourceGroup resourceGroup)
+            ResourceGroupResource resourceGroup)
         {
-            return await azure.AppServices.AppServicePlans
-                .Define(planName)
-                .WithRegion(Region.USWest3)
-                .WithExistingResourceGroup(resourceGroup)
-                .WithPricingTier(PricingTier.StandardS1)
-                .WithOperatingSystem(OperatingSystem.Windows)
-                .CreateAsync();
+            AppServicePlanCollection appServicePlans = resourceGroup
+                .GetAppServicePlans();
+
+            AppServicePlanData appServicePlanData =
```

Comment:
If the right side type is explicit then use `var` on the left.
For instance:
```
var student = new Student();
```

Don't do:
```
Student student = new Student();
```

Fix all others the same way